src/Features/Core/Portable/EncapsulateField/AbstractEncapsulateFieldService.cs

```diff
@@ -358,8 +358,10 @@ protected string GeneratePropertyName(string fieldName)
                 baseName = fieldName;
             }
 
-            // Make uppercase the first letter
-            return char.ToUpper(baseName[0]).ToString() + baseName.Substring(1);
+            // Make the first character upper case using the "en-US" culture.  See discussion at
+            // https://github.com/dotnet/roslyn/issues/5524.
+            var firstCharacter = CompletionRules.EnUSCultureInfo.TextInfo.ToUpper(baseName[0]);
```

Comment:
I spoke with @davkean offline and it was decided that to avoid churn in the `stabilization` branch this PR will be left as-is and I've filed issue #6442 to properly collect all usages of `new CultureInfo("en-US")`.